server/src/main/webapp/WEB-INF/rails/webpack/views/components/link/index.tsx

```diff
@@ -28,6 +28,7 @@ export interface Attrs {
   externalLinkIcon?: boolean;
   onclick?: (e: MouseEvent) => void;
   title?: string;
+  dataTestId?: string;
```

Comment:
what happens if instead of passing this attribute we set it at the top level?
Meaning, doing `<Link data-test-id="something" />` wouldn't set it?

Something similar to [this](https://github.com/gocd/gocd/blob/master/server/src/main/webapp/WEB-INF/rails/webpack/views/pages/new-environments/create_env_modal.tsx#L74-L75).

Reply:
I would prefer `dataTestId ` over `data-test-id` as it will provide suggestion in IDE. On top of that not all the components are copying over all attributes